foreigndonors.php

```diff
@@ -115,6 +115,8 @@ function foreigndonors_civicrm_buildForm($formName, &$form) {
 
     $isNSW = substr($finType[0]['financial_type_id:label'], 0, 3) === 'NSW';
     $isACT = substr($finType[0]['financial_type_id:label'], 0, 3) === 'ACT';
+    $form->assign('addNSWProhibitedDonor', $isNSW);
+    $form->assign('addACTProhibitedDonor', $isACT);
```

Comment:
@andrew-cormick-dockery don't you need to include the $domainId part of the if statement as well here?